bin/porta

```diff
@@ -432,31 +432,23 @@ module Porta
 
     protected
 
-    def docker_internal_host
-      options[:docker_internal_host] ||= container_local_host_ip
+    def podman_internal_host
+      options[:podman_internal_host] ||= container_local_host_ip
     end
 
     def container_local_host_ip(image: nil)
       gw_ip_cmd = %{printf "%d.%d.%d.%d" $(awk '$2 == 00000000 && $7 == 00000000 { for (i = 8; i >= 2; i=i-2) { print "0x" substr($3, i-1, 2) } }' /proc/net/route)}
-      `docker run --rm --entrypoint /bin/bash #{image || options[:porxy_image]} -c #{gw_ip_cmd.shellescape}`
+      `podman run --rm --entrypoint /bin/bash #{image || options[:porxy_image]} -c #{gw_ip_cmd.shellescape}`
     end
 
     def container_opts_local_hosts(*hosts)
-      # TODO: podman now wupports host.containers.internal hostname for this
-      #       without need for special --net option
-      #       see https://github.com/containers/podman/issues/10878
-      unless macos?
-        hosts << "host.docker.internal"
-        if `docker --version`.include?("podman")
-          gw_ip = docker_internal_host
-          host_access = "--net slirp4netns:allow_host_loopback=true,enable_ipv6=false"
-          if File.read("/etc/resolv.conf") =~ /nameserver\s+127\.0\.0\.1$/
-            host_access += " --dns=#{gw_ip}"
-          end
-        end
+      hosts << "host.containers.internal"
+      gw_ip = podman_internal_host || 'host-gateway'
+      if File.read("/etc/resolv.conf") =~ /nameserver\s+127\.0\.0\.1$/
+        hosts.map { |h| "--add-host=#{h}:#{gw_ip} " }.join + " --dns=#{gw_ip}"
+      else
+        hosts.map { |h| "--add-host=#{h}:#{gw_ip} " }.join
```

Comment:
Can we still keep docker support here? It's fine to keep everything podman by default but fully stop supporting docker is a little bit limiting I think.

Reply:
I would keep support for docker if it's not much complicated. If keeping docker makes this too messy I would drop it.

Reply:
My problems launching `apicast` and `apisonator_worker` due to network misconfiguration persist. I could fix them locally adding back the `--net slirp4netns:allow_host_loopback=true,enable_ipv6=false` CLI argument.